packages/tpetra/core/src/Tpetra_Details_unpackCrsGraphAndCombine_def.hpp

```diff
@@ -136,78 +137,69 @@ unpackRow(typename Kokkos::View<GO*,Device,Kokkos::MemoryUnmanaged>& gids_out,
 /// Data (bytes) describing the row of the CRS graph are "unpacked"
 /// from a single (concatenated) (view of) Packet* directly into the
 /// row of the graph.
-template<class Packet, class LocalGraph, class LocalMap, class BufferDevice>
+template<class LocalOrdinal, class Packet, class RowView,
+         class IndicesView, class Device, class BufferDevice>
 class UnpackAndCombineFunctor {
 
-  typedef Packet packet_type;
-  typedef LocalMap local_map_type;
-  typedef LocalGraph local_graph_type;
-  typedef BufferDevice buffer_device_type;
+  using LO = LocalOrdinal;
+  using GO = typename IndicesView::value_type;
+  using packet_type = Packet;
+  using row_ptrs_type = RowView;
+  using indices_type = IndicesView;
+  using buffer_device_type = BufferDevice;
 
-  typedef typename local_map_type::local_ordinal_type LO;
-  typedef typename local_map_type::global_ordinal_type GO;
   // Kokkos::parallel_reduce fails to compile if named device_type and typedef
```

Comment:
Is this comment still relevant?  I'm guessing not.  If not, now would be a good chance to remove it :-)

Reply:
👍